RecoLocalTracker/Configuration/python/RecoLocalTracker_EventContent_cff.py

```diff
@@ -7,6 +7,7 @@
     'keep DetIdedmEDCollection_siPixelDigis_*_*',
     'keep *_siPixelClusters_*_*', 
     'keep *_siStripClusters_*_*',
+    'keep *_siPhase2Clusters_*_*',
```

Comment:
I believe these would be better added only with `phase2_tracker` era, see e.g. https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/RecoLocalMuon/Configuration/python/RecoLocalMuon_EventContent_cff.py#L31.